code/modules/mod/modules/modules_maint.dm

```diff
@@ -350,6 +361,10 @@
 	if(you_fucked_up || mod.wearer.has_gravity() > NEGATIVE_GRAVITY)
 		return
 
+	// Prevent infinite loops when being fireman carried - just don't trigger at all
+	if(mod.wearer.buckled)
+		return
```

Comment:
Should probably stack trace - since we block all buckled, we can reasonably assert that this shouldn't happen

Or just call like `mod.wearer.unbuckle_all_mobs()` ?